djl-spring-boot-mxnet-autoconfigure/src/main/java/ai/djl/spring/configuration/EngineType.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance
+ * with the License. A copy of the License is located at
+ *
+ * http://aws.amazon.com/apache2.0/
+ *
+ * or in the "license" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES
+ * OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions
+ * and limitations under the License.
+ */
+package ai.djl.spring.configuration;
+
+public enum EngineType {
+    MXNET("MXNet"),
+    PYTORCH("PyTorch"),
```

Comment:
Are these supported at the moment? It is a great way to deal with the future, but as soon these properties show up, user will see them in auto-completion and can select it. 

Thinking more about this: I suppose we will need a separate starter for pytorch and the rest (e.g. just dependency packaging). I am fine keeping this auto-configuration generic (e.g. one for all types) but it needs to be augmented with documentation like: `if you use MXNET then make sure you have djl-spring-boot-starter-mxnet-auto` as dependency`. 

In the short run I would either recommend changing documentation and marking unsupported clearly as such, or just removing them for now, and adding those one we have support for them. The latter will be preferred since it will avoid redundant incidents of users running into issues.

Reply:
agree, I think we should just remove this.

Reply:
1. Removed EngineType, we don't need this in spring-boot use case.
2. Remove artifactId, this behavior will be changed in 0.4.0, add this property back in future release.